scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -604,7 +605,21 @@ public TerminationData taskTerminatedWithResult(TaskId taskId, TaskResultImpl re
                         return terminationData;
                     }
                 } else if (numberOfExecutionLeft <= 0) {
-                    jobData.job.increaseNumberOfFaultyTasks(taskId);
+                    if (!onErrorPolicyInterpreter.requiresPauseTaskOnError(task) &&
+                        !onErrorPolicyInterpreter.requiresPauseJobOnError(task) &&
+                        !onErrorPolicyInterpreter.requiresCancelJobOnError(task)) {
+                        jobData.job.increaseNumberOfFaultyTasks(taskId);
```

Comment:
I Think, we can read the Executions counter like this:
  * 7 is the total number of executions of the task  
  * 2 is the number of automatic restarts of the task by the scheduler
  * 7-2 is the number of manually restarts of the task by the user